common/terms.html

```diff
@@ -79,7 +79,19 @@
     <dd>
       <p>Translated to platform-specific <a class="termref" data-lt="accessibility api">accessibility APIs</a> as defined in the <a href="" class="core-mapping">Core Accessibility API Mappings</a>.</p>
     </dd>
-	<dt><dfn data-lt="graphical document|graphical documents">Graphical Document</dfn></dt>
+  </dd>
+  <dt><dfn>Focusable</dfn></dt>
+  <dd>
+    <p>An element or an area in a document is focusable, if it can receive focus. In different circumstances one can use one or more of the following methods to set focus:</p>
+    <ul>
+      <li>by scripting</li>
+      <li>using a mouse</li>
+      <li>using keyboard</li>
+      <li>by touch.</li>
+    </ul>
+    <p>For a detailed description see <a href="https://html.spec.whatwg.org/#focusable-area">Focusable area</a> in the HTML Specification.</p>
```

Comment:
```suggestion
    <p>An element or area matching the definition of <a href="https://html.spec.whatwg.org/#focusable-area">focusable area</a> in the HTML Specification.</p>
```

Reply:
Replying to @MarioBatusic's email response question with a specific edit suggestion. If the term is to be defined in ARIA, it should be a direct cross-reference to the other spec's definition, rather than a terse-but-incomplete variant.